### ISSUE TITLE
Creating areas from Collector should display current status of packaging

### ISSUE THREAD

**Issue by ananovosel** [doug-m/CollectorAuroraTesting#235]
### Issue
After one defines area to download in Collector, it is confusing if area is being packed or not. New area simply appears after some time.

### Expected Behavior
Collector should show some kind of status while creating areas on-demand.

### Device Information
* Device: Huawei P20
* OS Version: Android

**Comment by colinlawr** [comment]
Sounds like known issue Apps-Android 1331

**Comment by colinlawr** [comment]
@ananovosel In this scenario, when you press download, is it opening the map instead of indicating the area is downloading?

**Comment by DerekLaw** [comment]
I also had this experience. I tried several times to create a new offline area (see Offline Areas Download Fails #223 for details). In fact, I tried 3-4 times - thinking I was not successful. When I returned back to the office about 15 mins later, I noticed that when viewing the map in the "map list" dialog, there is some new text added below the map name "offline areas" - see screenshot below.

![Screenshot_20190808-102019](https://user-images.githubusercontent.com/1037228/62723993-d2a1e800-b9c6-11e9-8c23-5bd873d188bb.png)

When I click the map, my expectation is that I would see a list of the 3-4 offline areas I created, so I could select one of them. Instead, I am immediately taken to map display. Not sure, if this is expected?

**Comment by ananovosel** [comment]
> @ananovosel In this scenario, when you press download, is it opening the map instead of indicating the area is downloading?

@colinlawr No, map was not opening but it did not indicate download neither. It was like @DerekLaw described. I also thought creating Areas is not working, but they just appeared after some time. I tried again today and now I see Area starts to download as soon as I define it.
Strange, but it seems it is working now as expected.
